File: Client/tecmo-wager/src/app/core/models/pending-activation.model.ts
export interface PendingActivation {
  pendingActivationId: number;
  googleSubjectId: string;
  email: string;
  fullName: string;
  requestedProfilePic: number;
  status: string;
  requestedAt: string;
  activatedAt?: string | null;
  activatedByPlayerId?: number | null;
}

/** Non-deleted player with no Google id (admin link dropdown). */
export interface AdminPlayerLinkListItem {
  playerId: number;
  fullName: string;
  emailAddress: string;
}

File: Service/Models/PendingActivationModel.cs
using TecmoTourney;

namespace TecmoTourney.Models
{
    public class PendingActivationModel
    {
        public int PendingActivationId { get; set; }
        public string GoogleSubjectId { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string FullName { get; set; } = string.Empty;
        public int RequestedProfilePic { get; set; }
        public PendingActivationStatus Status { get; set; } = PendingActivationStatus.Pending;
        public DateTime RequestedAt { get; set; }
        public DateTime? ActivatedAt { get; set; }
        public int? ActivatedByPlayerId { get; set; }
    }
}
